bgpd/bgp_route.c

```diff
@@ -5800,17 +5800,27 @@ static void bgp_aggregate_install(struct bgp *bgp, afi_t afi, safi_t safi,
 			aggregate, atomic_aggregate, p);
 
 		if (!attr) {
+			char buf1[INET6_ADDRSTRLEN];
+			inet_ntop(p->family, &p->u.prefix, buf1, INET6_ADDRSTRLEN);
 			bgp_aggregate_delete(bgp, p, afi, safi, aggregate);
+			zlog_info("%s(%s): null attribute", __FUNCTION__, buf1);
```

Comment:
get rid of buf1 and use %pFX here please for p

Reply:
`__function__` => `__func__`

Reply:
and I think it should be zlog_debug here

Reply:
I have changed it to
zlog_info("%s(%pFX): null attribute", __FUNCTION__, p);
Also, I feel that we should keep this information as zlog_info() as it is an important log.

Reply:
I see that in some places in code, we still use __FUNCTION__. So it is fine. Thanks
bgpd/bgpd.h:1807:#define peer_unlock(A) peer_unlock_with_caller(__FUNCTION__, (A))
bgpd/bgpd.h:1808:#define peer_lock(B) peer_lock_with_caller(__FUNCTION__, (B))

Reply:
Please do what is being asked.  Also guard the zlog_debug with an appropriate debug check.

Reply:
The reason this is a debug:  What can an operator do with this information?  If there is nothing they can do telling them about it is not useful.

Reply:
Thanks. I have updated as per review comments.